app/src/main/kotlin/com/wire/android/ui/authentication/devices/remove/RemoveDeviceScreen.kt

```diff
@@ -75,7 +77,7 @@ private fun RemoveDeviceContent(
                 is RemoveDeviceState.Success ->
                     RemoveDeviceItemsList(lazyListState, state.deviceList, false, onItemClicked)
                 RemoveDeviceState.Loading ->
-                    RemoveDeviceItemsList(lazyListState, List(10) { Device() }, true, onItemClicked)
```

Comment:
This `List(10) { Device() }` was to show the shimmer effect while loading data

Reply:
ohhh nice catch, i will add it again

Reply:
@saleniuk Done!